### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by asclines**
This PR is first a place to discuss the changes to the issue labels.

Once the discussion is done, this PR can be be accepted so that the documentation is updated.

My updates to the issues are inspired by [this link](https://robinpowered.com/blog/best-practice-system-for-organizing-and-tagging-github-issues/) and before discussing any changes to the issue labels I recommend you read this first. 

I will do my best to relabel the currently open issues but I would recommend that you make sure the issues you are assigned to are properly labeled after this..

For those of you who still haven't clicked the link, this is the chart from that site that could act as a **tl;dr** for this.

[Quick link to labels](https://github.com/ASAAR/SE2-KaggleComp/labels)

So I have updated the labels, any suggestions? Comments? Concerns?

### CODE REVIEW

**Comment by asclines** [comment]
I hope that these feedback labels can really mainstream not only task completion but discussions as well.

With the `questions `label we can provide more clarity to an issue in a way that is clean and easy to reference in the future.
With the `discussions `label we can separate and preserve conversations that solve a particular problem and hash out any confusion.

Why these labels as opposed to slack?
With these conversations being public and with a specific topic in mind, not only might the conversation be more **productive** it separates out conversations based on topic so it will be easier to  come back to in the future and continue the conversation without interfering with another.

**Review by isaac-gs — APPROVED** [review]

**Comment by ASAAR** [comment]
Yeah I think we should also provide the definitions of each label color in our documentation, to make it more prominent. If we copy theirs we can just credit them (as you've done).

**Comment by ZakeryFyke** [comment]
I enjoy labels
Especially for team projects
This will be handy

**Comment by asclines** [comment]
@ASAAR I agree, should that be in the Contributing.md ?

**Comment by ASAAR** [comment]
@asclines Yeah, it should. I'll add that

**Comment by asclines** [comment]
@Everyone! When @ASAAR and I add label definitions, that would be the time for you to voice your opinion on them.

**Comment by ZakeryFyke** [comment]
Here is a haiku
To make up for the last one
Just for you old man

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,7 +9,6 @@ Please note we have a code of conduct, please follow it in all your interactions
 We try to stick with Google's code standards. Please call us on them if we don't stick to them.
 
 * [Python](https://google.github.io/styleguide/pyguide.html)
-* [Javascript](https://google.github.io/styleguide/javascriptguide.xml)
```
Comment (asclines) [inline CONTRIBUTING.md]: Why was this removed?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,7 +20,18 @@ We try to stick with Google's code standards. Please call us on them if we don't
    do not have permission to do that, you may request the second reviewer to merge it for you.
 
 ## Issue Labels  
-If you are a contributor to this repository it is your responsibility to make sure all issues are properly labeled.
+If you are a contributor to this repository it is your responsibility to make sure all issues are properly labeled. We are following [this](https://robinpowered.com/blog/best-practice-system-for-organizing-and-tagging-github-issues/) standard. As follows,
```
Comment (asclines) [inline CONTRIBUTING.md]: I wouldn't say we all following their standard so much as our standard was inspired by.. that way we can modify as we please

File: CONTRIBUTING.md (hunk 1 above)
Comment (ASAAR) [inline CONTRIBUTING.md]: If I did it, it was an accident. It should be there

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -20,6 +20,20 @@ We try to stick with Google's code standards. Please call us on them if we don't
 3. You may merge the Pull Request in once you have the sign-off of two other developers, or if you
    do not have permission to do that, you may request the second reviewer to merge it for you.
 
+## Issue Labels  
+If you are a contributor to this repository it is your responsibility to make sure all issues are properly labeled. We are following [this](https://robinpowered.com/blog/best-practice-system-for-organizing-and-tagging-github-issues/) standard. As follows,
```
Comment (asclines) [inline CONTRIBUTING.md]: I wouldn't say we all following their standard so much as our standard was inspired by.. that way we can modify as we please

Comment (ASAAR) [inline CONTRIBUTING.md]: Would you like me to add "somewhat" to the sentence?

Comment (asclines) [inline CONTRIBUTING.md]: Yeah that could work
